pkg/apis/gcp/v1alpha1/types_infrastructure.go

```diff
@@ -179,4 +181,7 @@ type InfrastructureState struct {
 	// Data is map to store things.
 	// +optional
 	Data map[string]string `json:"data,omitempty"`
+	// Routes contains information about cluster routes
+	// +optional
+	Routes []apisgcp.Route `json:"routes,omitempty"`
```

Comment:
For consistency I would mirror the struct here instead of importing the one from the internal package.